File: src/main/java/jp/artan/japanesefoodmod/raw_materials/wood/RawMaterialsFruitBlock.java
package jp.artan.japanesefoodmod.raw_materials.wood;

import jp.artan.japanesefoodmod.common.wood.FruitBlock;
import jp.artan.japanesefoodmod.init.CreativeTabsInit;

public class RawMaterialsFruitBlock extends FruitBlock {

    public RawMaterialsFruitBlock(String name) {
        super(name);
        this.setCreativeTab(CreativeTabsInit.fruitTreeCreativeTab);
    }
}

File: src/main/java/jp/artan/japanesefoodmod/raw_materials/wood/RawMaterialsFruitBlock.kt
package jp.artan.japanesefoodmod.raw_materials.wood

import jp.artan.japanesefoodmod.common.wood.FruitBlock
import jp.artan.japanesefoodmod.init.CreativeTabsInit

class RawMaterialsFruitBlock(name: String) : FruitBlock(name) {
    init {
        this.setCreativeTab(CreativeTabsInit.fruitTreeCreativeTab)
    }
}
